src/System.Net.Http/src/System/Net/Http/NetEventSource.Http.cs

```diff
@@ -65,6 +66,19 @@ public static void ClientSendCompleted(HttpClient httpClient, HttpResponseMessag
             WriteEvent(HandlerMessageId, handlerId, workerId, requestId, memberName, message);
             //Console.WriteLine($"{handlerId}/{workerId}/{requestId}: ({memberName}): {message}");  // uncomment for debugging only
 
+        [NonEvent]
+        public static void AuthenticationInfo(Uri uri, string message)
+        {
+            if (IsEnabled)
+            {
+                Log.AuthenticationInfo(uri?.ToString(), message);
+            }
+        }
+
+        [Event(SelectAuthenticationId, Keywords = Keywords.Debug, Level = EventLevel.Verbose)]
```

Comment:
Why is all the logging using EventLevel.Verbose?  We should split this up so that "errors" are using EventLevel.Error.  We currently don't do a good job at this in the current EventSource tracing for networking components.  But I'd like to see us move toward a better design on this.  It's easier to diagnose "errors" filtering on EventLevel.Erorr separate from the other output that uses EventLevel.Verbose.

Reply:
Sounds like a useful change. I'll go ahead and update the rest of the file to be more accurate here too.